gdx/src/com/badlogic/gdx/Net.java

```diff
@@ -175,8 +176,15 @@ public void setUrl (String url) {
 		 * @param name the name of the header.
 		 * @param value the value of the header. */
 		public void setHeader (String name, String value) {
-			headers.put(name, value);
+                    List<String> val = new ArrayList();

```

Comment:
Oh, this is intended to replace any previously set value, never mind that last comment in that case.